### PR TITLE
Added Linear Search [Java]

### LINKED ISSUE #122: Linear Search [Java]
https://en.wikipedia.org/wiki/Linear_search

### PR DESCRIPTION
**Pull request by PiyushPawar17**
fixed #122 

#### By submitting this pull request I confirm I've read and complied with the below requirements.

- [x] I have read the [Contribution guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [x] I have followed the [coding guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#cs) for this project.
- [x] My code follows the [skeleton code structure](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#sample).
- [x] This pull request has a descriptive title. For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.

### CODE REVIEW

File: linear_search/LinearSearch.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+	public static int linearSearch(int[] arr, int searchElement) {
+		for(int i = 0; i < arr.length; i++){
+			if(arr[i] == searchElement)
+				return i;	// Element found
+		}
+		return -1;	// ELement not found
+	}
+
+	public static void main(String args[]) {
```
Comment (Monal5031) [inline linear_search/LinearSearch.java]: `[]` with type descriptors only, as `String[] args`

File: linear_search/LinearSearch.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/*
+*  Linear Search algorithm searches for a given number in array
+*  Time complexity is O(n) , n is size of array
+*/
+
+public class LinearSearch {
+
+	public static int linearSearch(int[] arr, int searchElement) {
+		for(int i = 0; i < arr.length; i++){
```
Comment (aashutoshrathi) [inline linear_search/LinearSearch.java]: space between ) and {

File: linear_search/LinearSearch.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+	}
+
+	public static void main(String[] args) {
+		int[] array = new int[100];
+		// Initializing array with 1,2,....,100
+		for(int i = 1; i <= 100; i++)
+			array[i-1] = i;
+		// Elements to be searched in the array
+		int[] search = {12,55,34,102,78};
+		for(int i = 0; i < 5; i++){
```
Comment (aashutoshrathi) [inline linear_search/LinearSearch.java]: add space between ) and {

File: linear_search/LinearSearch.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+*/
+
+public class LinearSearch {
+
+	public static int linearSearch(int[] arr, int searchElement) {
+		for(int i = 0; i < arr.length; i++){
+			if(arr[i] == searchElement)
+				return i;	// Element found
+		}
+		return -1;	// ELement not found
```
Comment (aashutoshrathi) [inline linear_search/LinearSearch.java]: change `ELement` to `element` 😆

File: linear_search/LinearSearch.java [hunk 5]
```diff
@@ -0,0 +1,31 @@
+/*
+*  Linear Search algorithm searches for a given number in array
+*  Time complexity is O(n) , n is size of array
+*/
+
+public class LinearSearch {
+
```
Comment (aashutoshrathi) [inline linear_search/LinearSearch.java]: is this blank line needed ? @Monal5031

Comment (aashutoshrathi) [inline linear_search/LinearSearch.java]: @PiyushPawar17 Is this blank line needed ?

**Pull request merged**